src/Engine/Graphics/BspRenderer.cpp

```diff
@@ -6,32 +6,56 @@
 
 #include "Library/Logger/Logger.h"
 
+// TODO(yoctozepto): we should not see it here
 BspRenderer *pBspRenderer = new BspRenderer();
 
-//----- (004B0EA8) --------------------------------------------------------
-void BspRenderer::AddFaceToRenderList_d3d(int node_id, int uFaceID) {
-    int pTransitionSector;  // ax@11
-    // int dotdist;                              // edx@15
 
-    if (uFaceID >= pIndoor->pFaces.size())
-        return;
```

Comment:
I think this should go to an assert with a "please create issue / provide save game" message until we are 100% all is well

Reply:
Makes sense.